sway-core/src/asm_lang/virtual_ops.rs

```diff
@@ -403,10 +413,10 @@ impl VirtualOp {
             LOGD(_r1, _r2, _r3, _r4) => vec![],
             MINT(_r1) => vec![],
             RVRT(_r1) => vec![],
-            SRW(r1, _r2) => vec![r1],
-            SRWQ(_r1, _r2) => vec![],
-            SWW(_r1, _r2) => vec![],
-            SWWQ(_r1, _r2) => vec![],
+            SRW(r1, r2, _r3) => vec![r1, r2],
```

Comment:
The comment above this method states:

> /// Returns a list of all registers *written* by instruction `self`. All of our opcodes define
> /// exactly 0 or 1 register, so the size of this returned vector should always be at most 1.

Now that SRW writes to rA and rB, will this break any assumptions in the compiler?

Reply:
I think we should be fine. We iterate over the result of `def_registers()` everywhere. I don't believe there are any baked assumptions, so this comment can probably be updated.